app/controllers/api/base_controller.rb

```diff
@@ -10,6 +10,56 @@ class BaseController < ActionController::API
     rescue_from ArgumentError, with: :bad_request_response
     rescue_from API::Errors::FilterValidationError, with: :filter_validation_error_response
 
+    def append_info_to_payload(payload)
```

Comment:
is this something we can move into a concern, and could we add a test for this as well please?

Reply:
Moved to concern and added test